### PR TITLE
[frontend-api] JwtConfiguration is now created on demand

### PR DESCRIPTION
**Pull request by TomasLudvik**
#### Description, the reason for the PR
During run of simple tasks like checking standards there is no need to have frontend-api keys generated, but current situation required it so this PR changes such behavior and \Lcobucci\JWT\Configuration is not created until it is needed.

#### Fixes issues
... 

#### Check the appropriate checkboxes below, please

- [ ] [BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/) 
- [ ] New feature 
- [x] I have read and signed [License Agreement for contributions](https://www.shopsys.com/license-agreement)

----
:globe_with_meridians: Live Preview:
  - https://tl-jwt-configuration-fix.odin.shopsys.cloud
  - https://cz.tl-jwt-configuration-fix.odin.shopsys.cloud

### CODE REVIEW

File: packages/frontend-api/src/Model/Token/JwtConfigurationFactory.php [hunk 1]
```diff
@@ -29,15 +31,23 @@ public function __construct(
      */
     public function create(): Configuration
     {
+        if (isset($this->configuration)) {
```
Comment (vitek-rostislav) [inline packages/frontend-api/src/Model/Token/JwtConfigurationFactory.php]: why do not you use `InMemoryCache` service for the caching?

File: packages/frontend-api/src/Model/Token/JwtConfigurationFactory.php [hunk 2]
```diff
@@ -14,6 +14,8 @@ class JwtConfigurationFactory
 {
     protected const FRONTEND_API_KEYS_FILEPATH_PARAMETER = 'shopsys.frontend_api.keys_filepath';
 
+    protected Configuration $configuration;
```
Comment (vitek-rostislav) [inline packages/frontend-api/src/Model/Token/JwtConfigurationFactory.php]: the class should not be defined as service anymore, right? So the following lines should be removed from the `services.yaml`:

```
    Lcobucci\JWT\Configuration:
        factory: [ '@Shopsys\FrontendApiBundle\Model\Token\JwtConfigurationFactory', create ]
```

Comment (vitek-rostislav) [reply packages/frontend-api/src/Model/Token/JwtConfigurationFactory.php]: btw, the change deserves upgrade notes IMHO (something like "Lcobucci\JWT\Configuration is not a service anymore, use Shopsys\FrontendApiBundle\Model\Token\JwtConfigurationFactory::create() to get the configuration")

**Review by vitek-rostislav — CHANGES_REQUESTED** [review]

File: packages/frontend-api/src/Model/Token/JwtConfigurationFactory.php (hunk 1 above)
Comment (TomasLudvik) [reply packages/frontend-api/src/Model/Token/JwtConfigurationFactory.php]: I do not want to use ResetInterface for this as it is not necessary

**Review by vitek-rostislav — APPROVED** [review]

**Review by vitek-rostislav — APPROVED** [review]

**Review by pk16011990 — APPROVED** [review]

**Pull request merged**